docs/API_ANAHTARI_REHBERI.md

```
@@ -0,0 +1,100 @@
+### 2. API Anahtarı Alma
+1. ElevenLabs hesabınıza giriş yapın
+2. Sağ üst köşedeki profil menüsüne tıklayın
+3. "Profile Settings" seçin
+4. "API Key" sekmesine gidin
+5. "Generate New API Key" butonuna tıklayın
+6. API anahtarınızı kopyalayın (xi-api-key-... formatında olacak)
+
+### 3. API Anahtarını Ayarlama
+
+#### Frontend (.env dosyası)
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)